patches/devtools-frontend/fix_explicitly_specify_typeroots.patch

```diff
@@ -0,0 +1,43 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Andy Dill <andy.dill@gmail.com>
+Date: Thu, 13 Feb 2020 20:06:42 -0800
+Subject: fix: explicitly specify `typeRoots`
+
+fix: explicitly specify `typeRoots` to avoid crawling parent directories
```

Comment:
This looks to be upstreamable, can you send a CL and link it in this PR ? Thanks!

Reply:
I've been out on PTO, but I'm back. I haven't contributed to any Chromium-related projects yet so this might be a bit of a process but I will get the ball rolling and link when I've got a CL submitted.

Reply:
Thanks! incase it helps https://github.com/ChromeDevTools/devtools-frontend#standalone-workflow

Reply:
Well, that was an adventure. https://chromium-review.googlesource.com/c/devtools/devtools-frontend/+/2070561

Reply:
Thanks! please add this link to the patch file description and should be good to merge this PR.